modules/storages/spec/services/storages/one_drive_managed_folder_sync_service_spec.rb

```diff
@@ -48,8 +48,31 @@
            origin_user_id: "33db2c84-275d-46af-afb0-c26eb786b194")
     s
   end
+  shared_let(:oidc_provider) { create(:oidc_provider) }
 
   # USER FACTORIES
+  shared_let(:oidc_user) do
+    identity_url = "#{oidc_provider.slug}:qweqweqweqwe"
+    create(:user, identity_url:)
+  end
+  shared_let(:oidc_admin) do
+    identity_url = "#{oidc_provider.slug}:zxczxczxczxc"
+    create(:admin, identity_url:)
+  end
+  shared_let(:oidc_user_token) do
+    create(:remote_identity,
```

Comment:
Why do we create a `remote_identity` but name it `*_token`?

For a variable called `oidc_user_token` I would have expected something that is a `OpenIDConnect::UserToken`, but not a `RemoteIdentity`.